libraries/AP_AHRS/AP_AHRS_DCM.h

```diff
@@ -122,8 +113,11 @@ class AP_AHRS_DCM : public AP_AHRS {
     bool pre_arm_check(bool requires_position, char *failure_msg, uint8_t failure_msg_len) const override;
 
 private:
-    float _ki;
-    float _ki_yaw;
+
+    // these are experimentally derived from the simulator
+    // with large drift levels
+    static constexpr float _ki{0.0087f};
```

Comment:
Const in capital?

Reply:
That would mean changing all the places it is used....